web/src/SecureVote/Components/UI/Collapsible.elm

```diff
@@ -0,0 +1,28 @@
+module SecureVote.Components.UI.Collapsible exposing (..)
+
+import Element exposing (..)
+import Element.Attributes exposing (..)
+import Element.Events exposing (onClick)
+import SecureVote.Components.UI.CommonStyles exposing (..)
+
+
+collapsible w { onCollapse, isCollapsed, header, body } =
+    let
+        collapsedSymbol =
+            if isCollapsed then
+                "➕"
+            else
+                "➖"
+
+        collapsedSection =
+            if isCollapsed then
+                []
+            else
+                body
+    in
+    column (w Collapsible)
+        [ spacing cmdSpacing ]
+    <|
+        [ el (w Title) [ onClick onCollapse, vary BtnCursor True ] <| text <| collapsedSymbol ++ " " ++ header
+        ]
+            ++ collapsedSection
```

Comment:
I found a cool new function the other day. You can replace this with just 

```
[ el (w Title) ...
, when isCollapsed body
]
```

http://package.elm-lang.org/packages/mdgriffith/style-elements/4.3.0/Element#when

Reply:
👍 awesome - good find